pliers/extractors/models.py

```diff
@@ -84,3 +84,65 @@ def _extract(self, stim):
             values.append(float(extraction[1]))
 
         return ExtractorResult([values], stim, self, features=features)
+
+
+def _resize_image(image, shape):
+    import numpy as np
+    from PIL import Image
+    return np.array(
+        Image.fromarray(image).resize(shape, resample=Image.BILINEAR))
+
+
+class TensorFlowKerasInceptionV3Extractor(ImageExtractor):
+
+    ''' Labels objects in images using a pretrained Inception V3 architecture
```

Comment:
Maybe add a note that images must be 299x299x3, and will be resampled (with distortion) if a different input is passed. That way at least people are aware that they might get wonky results unless they do some editing.